### ISSUE TITLE
[Discussion] If we remove validation Module CR in CLI

### ISSUE THREAD

**Issue by ruanxin** [kyma-project/cli#1846]
At the moment, we introduce envTest into CLI to do module CR validation.

Cons:
- It introduces a test framework into production code, which is wired, also takes time to provision a in-memory cluster to do validation.
- To make the log work as expected, it must initialize a different logger, can't use log from CLI. https://github.com/kyma-project/cli/issues/1763

Does this validation bring value? @janmedrek 
Basically, this validation can verified in Module Teams' testing pipeline or module submission pipeline, is it really worth this effort to do validation during module template creation?

If have benefit from user side, we can think some static validation implementation:
https://github.com/kubernetes/apiextensions-apiserver/blob/v0.28.3/pkg/apiserver/validation/validation.go

**Comment by tobiscr** [comment]
Not sure if we need this feature. We got the feedback in teh past, that teams have to ensure that their parts are correct. Maybe we can drop this step completely and leave it in team's responsibility?

**Comment by janmedrek** [comment]
I agree with @tobiscr, we should drop it completely.

**Comment by janmedrek** [comment]
The final decision:
- drop all the validation
- remove dependency to the `envtest`
